syntax = "proto3";

message PersonCreation {
  string first_name = 1;
  string last_name = 2;
  string company_name = 3;
}

message PersonCreated {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

service PersonService {
  rpc create(PersonCreation) returns (PersonCreated);
}
